pkg/controller/migmigration/task.go

```diff
@@ -60,68 +64,97 @@ const (
 	HasVerify    = 0x08 // Only when the plan has enabled verification
 )
 
-type Itinerary []Step
+type Itinerary struct {
+	Name  string
+	Steps []Step
+}
 
 var StageItinerary = Itinerary{
-	{phase: Created},
-	{phase: Started},
-	{phase: Prepare},
-	{phase: EnsureCloudSecretPropagated},
-	{phase: AnnotateResources, all: HasPVs},
-	{phase: EnsureStagePods, all: HasPVs},
-	{phase: StagePodsCreated, all: HasStagePods},
-	{phase: RestartRestic, all: HasStagePods},
-	{phase: ResticRestarted, all: HasStagePods},
-	{phase: QuiesceApplications, all: Quiesce},
-	{phase: EnsureQuiesced, all: Quiesce},
-	{phase: EnsureStageBackup, all: HasPVs},
-	{phase: StageBackupCreated, all: HasPVs},
-	{phase: EnsureStageBackupReplicated, all: HasPVs},
-	{phase: EnsureStageRestore, all: HasPVs},
-	{phase: StageRestoreCreated, all: HasPVs},
-	{phase: EnsureStagePodsDeleted, all: HasStagePods},
-	{phase: EnsureStagePodsTerminated, all: HasStagePods},
-	{phase: EnsureAnnotationsDeleted, all: HasPVs},
-	{phase: Completed},
+	Name: "StageItinerary",
```

Comment:
Please let's not include `Itinerary` as part of name.  It's redundant and results in:
```
Status:
  itinerary: "StageItinerary"
```